### DISCUSSION TITLE
[Question] Start lima with read or write host directory

### DISCUSSION THREAD

**Discussion by jhay06** [lima-vm/lima#1137] [Q&A] [answered]
Is this posible ? if yes how ? 
Im currently running lima on Ventura and when i try to run touch a on the current directory it says argument dest: the destination . is not write-able at /Users/me/gits/test

how can i make it writable

**Comment by AkihiroSuda** [comment, marked as answer]
You can run `limactl edit` to open the YAML editor, and set `writable` to `true`

https://github.com/lima-vm/lima/blob/680691c746f58638dcc7e0ee1370745a597af77c/examples/default.yaml#L45-L53
